deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -844,7 +844,7 @@ bool IsolateShim::RunSingleStepOfReverseMoveLoop(v8::Isolate* isolate,
     int64_t snapEventTime = -1;
     int64_t snapEventEndTime = -1;
     int64_t origNETime = *nextEventTime;
-    JsTTDMoveMode _moveMode = (JsTTDMoveMode)(*moveMode);
+    JsTTDMoveMode _moveMode = (JsTTDMoveMode)(*moveMode);  // TODO JsTTDMoveMode is no longer 64 bit wide, fix use of moveMode parameter
```

Comment:
@mrkmarron I saw in your work that you left this cast as is, and did not change the size of the moveMode parameter.  Do you think it makes sense to change the parameter to a 32 bit size or is there still a need for it to be 64 bit?  I did not see an obvious need for the extra bits while examining the code.